pkg/certificate/mrc_reconciler.go

```diff
@@ -153,6 +159,33 @@ func (m *Manager) updateMRCComponentStatus(mrc *v1alpha2.MeshRootCertificate) er
 	return m.mrcClient.UpdateMeshRootCertificate(mrc)
 }
 
+// checks if the given MRC meets the attributes to move the componentStatuses to Validating
+func shouldChangeStatusToValidating(mrc *v1alpha2.MeshRootCertificate) bool {
+	// case-insensitive check for if strings are equal
+	if strings.EqualFold(string(mrc.Spec.Intent), constants.MRCIntentPassive) {
+		acceptedCond := getCondition(mrc, constants.MRCConditionTypeAccepted)
+
+		if strings.EqualFold(string(acceptedCond.Status), constants.MRCConditionStatusTrue) {
+			validRolloutCond := getCondition(mrc, constants.MRCConditionTypeValidatingRollout)
+
+			if strings.EqualFold(string(validRolloutCond.Status), constants.MRCConditionStatusFalse) && strings.EqualFold(validRolloutCond.Reason, constants.MRCConditionReasonPending) {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+// returns the MeshRootCertificateCondition that matches the condtion type specified
+func getCondition(mrc *v1alpha2.MeshRootCertificate, condType v1alpha2.MeshRootCertificateConditionType) *v1alpha2.MeshRootCertificateCondition {
```

Comment:
is there exactly 1 condition at a given condition type? If not, I'd consider renaming this func, or changing its signature to refelect that

Reply:
Yes there should not be multiple statuses of the same type

Reply:
gotcha, and is there only 1 condition type with status set to True at any given time?

Reply:
No there may be multiple types set to True at the same time